tests/global_config.py

```diff
@@ -436,6 +436,33 @@ def _get_default_storage_class(sc_list):
     },
 ]
 
+instance_type_centos_fedora_os_matrix = [
```

Comment:
should we keep centos fedora seperate?

Reply:
Iv'e considered this, but I feel like it's a waste of repeated code since they are very similar.

Reply:
i agree with @geetikakay

keep it simple;
you also have `@pytest.mark.usefixtures("xfail_if_centos")` wchih literally runs only on a subset of the matrix.